### ISSUE TITLE
JF-17 lua parsing error

### ISSUE THREAD

**Issue by wrycu** [pydcs/dcs#26]
The newly released JF-17 payload file cannot be parsed by pydcs because they don't include keys on each payload entry.  I haven't the faintest idea if this is valid LUA or not, but it's what was shipped with the module.

>  File "C:\Users\user\PycharmProjects\dcs_liberation\venv3_7\lib\site-packages\dcs\lua\parse.py", line 170, in object
>    raise se
>  File "<string>", line 4
>    Expected character '[', got '{'

Original payload entry:
```lua
local unitPayloads = {
    ["name"] = "JF-17",
    ["payloads"] = {
		{
			["name"] = "PL-5Ex2, C802AKx2, 800L Tank",
			["pylons"] = {
				[1] = {
					["CLSID"] = "DIS_C-802AK",
					["num"] = 5,
				},
				[2] = {
					["CLSID"] = "DIS_TANK800",
					["num"] = 4,
				},
				[3] = {
					["CLSID"] = "DIS_C-802AK",
					["num"] = 3,
				},
				[4] = {
					["CLSID"] = "DIS_PL-5EII",
					["num"] = 7,
				},
				[5] = {
					["CLSID"] = "DIS_PL-5EII",
					["num"] = 1,
				},
				[6] = {
					["CLSID"] = "DIS_C-701IR",
					["num"] = 6,
				},
				[7] = {
					["CLSID"] = "DIS_C-701IR",
					["num"] = 2,
				},
			},
			["tasks"] = {
				[1] = 30,
			},
		},
    },
    ["unitType"] = "JF-17",
}
return unitPayloads
```

Fixed payload entry (note the addition of `[1]` on line 4):
```lua
local unitPayloads = {
    ["name"] = "JF-17",
    ["payloads"] = {
		[1] = {
			["name"] = "PL-5Ex2, C802AKx2, 800L Tank",
			["pylons"] = {
				[1] = {
					["CLSID"] = "DIS_C-802AK",
					["num"] = 5,
				},
				[2] = {
					["CLSID"] = "DIS_TANK800",
					["num"] = 4,
				},
				[3] = {
					["CLSID"] = "DIS_C-802AK",
					["num"] = 3,
				},
				[4] = {
					["CLSID"] = "DIS_PL-5EII",
					["num"] = 7,
				},
				[5] = {
					["CLSID"] = "DIS_PL-5EII",
					["num"] = 1,
				},
				[6] = {
					["CLSID"] = "DIS_C-701IR",
					["num"] = 6,
				},
				[7] = {
					["CLSID"] = "DIS_C-701IR",
					["num"] = 2,
				},
			},
			["tasks"] = {
				[1] = 30,
			},
		},
    },
    ["unitType"] = "JF-17",
}
return unitPayloads
```

**Comment by rp-** [comment]
fixed in 94fe033aba2d0bebab1306e3fb0fce542adbedc2
and released 0.9.6

**Comment by wrycu** [comment]
Thanks for the quick turnaround.
